src/lib/core/components/visualizations/implementations/HistogramVisualization.tsx

```diff
@@ -824,13 +827,18 @@ function HistogramViz(props: VisualizationProps<Options>) {
     </>
   );
 
+  // set Undo icon props
+  const undoSize = 20;
+  const undoColor = '#006699';
```

Comment:
It would be good to start using the colors from the theme.

`core/components/FilterChip.tsx` is an example of how to do it.

I would put the undo button into its own component and then all the (minor) theme-related boilerplate is only needed once.

Reply:
I tried to make it as a component, ResetButton. However, I have made four examples with Boxplot Viz to hear your opinion (https://github.com/VEuPathDB/web-eda/pull/1401/commits/fb3925d0d439b5f87e4afadb84fe1b8ee035cd92), so it would be great if you can test them. Basically it is comprised of three approaches, CoreUI, MUI (Material-UI), and original (previous one using pure button/Tooltip). In the following screenshot there are four kinds of button icons starting from CoreUI (icon only - without text), CoreUI (icon with text), MUI, and original. CoreUI one has mouse hover event (you can test it), and somehow icon with text is not that bad. One thing I don't like for the first one, CoreUI - icon only without text, is that it looks like its tooltip size depends on the button size so the tooltip text, Reset to defaults, are shown in the three lines: the others are okay.

It would be great if you can share your thoughts on which one would be best fit to our purpose.

![boxplot-undo1](https://user-images.githubusercontent.com/12802305/195232682-1a7a14fe-a61a-4064-a15d-c0520b89f1c8.png)